app/assets/stylesheets/layouts/_l-single-article.scss

```diff
@@ -0,0 +1,49 @@
+.l-single-article {
+
+  &.-publications {
+    background-color: rgba($color-3, .04);
+  }
+
+  &.-activities {
+    background-color: rgba($color-1, .08);
+  }
+
+  &.-news {
+    background-color: rgba($color-2, .08);
+  }
+
+  .l-main-content {
+    background-color: inherit;
+  }
+
+  .image {
```

Comment:
Is not possible to add this image as background to the layout instead of creating a new div?